br/point-in-time-recovery.md

```diff
@@ -0,0 +1,93 @@
+
+When you run the `br restore point` command, you need to specify the latest snapshot backup data before the time point you want to restore and specify the log backup data. BR first restores the snapshot data, and then reads the log backup data between the snapshot time point and the specified restoration time point.
+
+### Manage backup data
+
+To manage backup data for PITR, you need to design a backup directory structure to store your backup data and regularly delete outdated or no longer needed backup data.
+
+- Organize the backup data in the following structure:
+
+    - Store the snapshot backup and log backup in the same directory for unified management. For example, `backup-${cluster-id}`.
+    - Store each snapshot backup in a directory whose name includes the backup date. For example, `backup-${cluster-id}/snapshot-20220512000130`.
+    - Store the log backup in a fixed directory. For example, `backup-${cluster-id}/log-backup`.
+
+- Delete the outdated or no longer needed backup data:
+
+    - When you delete the snapshot backup, you can delete the directory of snapshot backup.
+    - To delete the log backup before a specified time point, run the `br log truncate` command.
+
+## Capabilities
+
+- PITR log backup has a 5% impact on the cluster.
+- When you run log backup and snapshot backup at the same time, it has a less than 20% impact on the cluster.
+- On each TiKV node, PITR can restore snapshot data at 280 GB/h and log data at 30 GB/h.
+- With PITR, the RPO of disaster recovery is less than 20 minutes. Depending on the data size to be restored, the RTO varies from several minutes to several hours.
+- BR deletes outdated log backup data at a speed of 600 GB/h.
+
+> **Note:**
+>
+> - The preceding functional specification is based on test results from the following two testing scenarios. The actual data may be different.
+> - Snapshot data restoration speed = Snapshot data size / (duration * the number of TiKV nodes)
+> - Log data restoration speed = Restored log data size / (duration * the number of TiKV nodes)
+
+Testing scenario 1 (on [TiDB Cloud](https://tidbcloud.com)):
+
+- The number of TiKV nodes (8 core, 16 GB memory): 21
+- The number of Regions: 183,000
+- New log created in the cluster: 10 GB/h
+- Write (insert/update/delete) QPS: 10,000
+
+Testing scenario 2 (on-premises):
+
+- The number of TiKV nodes (8 core, 64 GB memory): 6
+- The number of Regions: 50,000
+- New log created in the cluster: 10 GB/h
+- Write (insert/update/delete) QPS: 10,000
+
+## Limitations
+
+- A single cluster can only start one log backup task.
+- You can only restore data to an empty cluster. To avoid impact on the services and data of the cluster, you cannot perform PITR in-place or on a non-empty cluster.
```

Comment:
```suggestion
- You can only restore data to an empty cluster. To avoid impact on the services and data of the cluster, do not perform PITR in-place or on a non-empty cluster.
```